docs/source/roadmap/receptor_roadmap_index.rst

```diff
@@ -0,0 +1,32 @@
+=================
+receptor Roadmaps
+=================
+
+The ``receptor`` team develops a roadmap for each major  ``receptor`` release.
+The latest roadmap shows current work; older roadmaps provide a history of the
+project.  We don't publish roadmaps for minor or subminor versions. So 2.0 and
+3.0 have roadmaps, but 2.1.0 and 2.10.1 do not.
+
+We incorporate team and community feedback in each roadmap, and aim for further
+transparency and better inclusion of both community desires and submissions.
+
+Each roadmap offers a *best guess*, based on the ``receptor`` team's experience
+and on requests and feedback from the community, of what will be included in a
+given release.  However, some items on the roadmap may be dropped due to time
+constraints, lack of community maintainers, and so on.
+
+Each roadmap is published both as an idea of what is upcoming in ``receptor``,
+and as a medium for seeking further feedback from the community.
+
+You can submit feedback on the current roadmap in multiple ways:
+- Create issue on github in ansible/receptor
+
+See `Ansible communication channels <https://github.com/ansible/ansible-documentation/blob/devel/docs/docsite/rst/community/communication.rst>`_ for details on how to
+join and use the email lists and chat channels.
```

Comment:
Consider if the receptor project wants to use [the forum](https://forum.ansible.com/) instead of email lists for discussions. You can request a dedicated forum group for receptor for example to help coordinate discussions in the project category.

Reply:
Nice idea, added